pymc3/distributions/dist_math.py

```diff
@@ -346,3 +349,134 @@ def grad(self, inputs, grads):
         x_grad, = grads
 
         return [x_grad * self.grad_op(x)]
+
+
+class Eigh(tt.nlinalg.Eig):
```

Comment:
Is this class here to fix the `njobs` issue mentioned on the discourse thread?

Reply:
Yes, it seems to be working.

Reply:
Is there a theano PR for this? Would be good to link to it.

Reply:
I just started one [here](https://github.com/Theano/Theano/pull/6557).